### PR TITLE
Update Test.java

### PR DESCRIPTION
**Pull request by WonJoonoThomasChoi**
asd

### CODE REVIEW

**Comment by WonJoonoThomasChoi** [comment]
asdasd

**Comment by 724thomas** [comment]
asdfasd

File: src/Test.java [hunk 1]
```diff
@@ -3,5 +3,6 @@ public static void main(String[] args) {
         System.out.println("Hello world!");
         System.out.println("Hello world!");
         System.out.println("Hello world!");
+        System.out.println("Hello world!");
     }
```
Comment (724thomas) [inline src/Test.java:7]: gdgd

File: src/Test.java [hunk 2]
```diff
@@ -3,5 +3,6 @@ public static void main(String[] args) {
         System.out.println("Hello world!");
         System.out.println("Hello world!");
```
Comment (724thomas) [inline src/Test.java:4]: fdfd

**Review by 724thomas — APPROVED** [review]

**Pull request merged**